pkgs/by-name/ve/vesktop/package.nix

```diff
@@ -0,0 +1,62 @@
+{ buildNpmPackage, fetchFromGitHub, vencord, electron, pipewire, lib }:
```

Comment:
nixpkgs-fmt?

Reply:
nixpkgs-fmt doesn't reformat anything here

Reply:
hmm. I would expect it to do something like this:
```suggestion
{ lib
, buildNpmPackage
, fetchFromGitHub
, vencord
, electron
, pipewire
}:
```